src/util.cpp

```diff
@@ -223,6 +224,14 @@ inline int OutputDebugStringF(const char* pszFormat, ...)
             static boost::mutex mutexDebugLog;
```

Comment:
I would have, but it's already here @mitchellcash. We've already got that part in for some reason (Yay for serialised logs!)